server/src/main/java/org/elasticsearch/index/mapper/FieldMapper.java

```diff
@@ -376,9 +371,8 @@ protected void doXContentBody(XContentBuilder builder, boolean includeDefaults,
 
         boolean indexed =  fieldType().indexOptions() != IndexOptions.NONE;
         boolean defaultIndexed = defaultFieldType.indexOptions() != IndexOptions.NONE;
-        if (includeDefaults || indexed != defaultIndexed ||
-            fieldType().tokenized() != defaultFieldType.tokenized()) {
```

Comment:
This is another tokenized() method than the one deleted above, no? I don't know enough about the details of how mappings work, but how are they different? How is this removal in the if-clause related to the rest of this change?